src/rard/utils/text_processors.py

```diff
@@ -1,11 +1,23 @@
 import re
 import string
+import unicodedata
 
 from django.utils.html import strip_tags
 
 
+def strip_combining(content):
+    """Converts the content to their base and combining characters,
+    then removes the combining ones and returns a lowercase string of the base characters
+    """
+    normalized = unicodedata.normalize("NFD", content)
+    return "".join(
+        [char for char in normalized if not unicodedata.combining(char)]
+    ).casefold()
```

Comment:
I think casefold is the cause of our failing test. Probably best to just remove it.